package.json

```diff
@@ -34,7 +34,11 @@
     "bluebird": "^3.1.1",
     "lodash": "^4.17.15",
     "semver": "^7.0.0",
-    "source-map-support": "^0.x"
+    "source-map-support": "^0.x",
+    "asyncbox": "^2.9.2",
+    "uuid": "^8.3.2",
+    "bplist-creator": "0.1.0",
```

Comment:
so we really want to stick to particular versions of these modules? We usually allow everything below the next major version, e.g.: `"bplist-creator": "^0.x",`